CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 2.8.11)
```

Comment:
One more thing: Would you mind adding a copyright notice at the top? (I'm not going to ask you to relinquish copyright, but if you happen to want to assign this to the public domain to match the other makefiles, explicitly state that instead.)

Reply:
Absolutely. I did not know what your policy for a copyright header was. I added a header without your name/web link cause it feels weird for me to assign copyright forcefully to someone :-).
I took the header from one of your .c files.

---
In reply to: [324059643](https://github.com/jamesderlin/dropt/pull/4#discussion_r324059643) [](ancestors = 324059643)

Reply:
Yeah, if I get more contributions I should add a `CONTRIBUTING` file with instructions and maybe an `AUTHORS` file.  For now I think you should add a copyright notice with your name (unless you want to reassign it).

Reply:
I put your name in there. I do not mind re-assigning it :-).

---
In reply to: [324295001](https://github.com/jamesderlin/dropt/pull/4#discussion_r324295001) [](ancestors = 324295001)

Reply:
Okay.  However, be aware that if you assign the copyright to me, there's a good chance that I will assign this particular file to the public domain later.  You may wish to retain copyright if you're not okay with that.

Reply:
I'm good as it is, if you want to later re-assign it that is fine.